etcdctl/ctlv3/command/printer.go

```diff
@@ -233,11 +233,12 @@ func makeEndpointStatusTable(statusList []epStatus) (hdr []string, rows [][]stri
 }
 
 func makeEndpointHashKVTable(hashList []epHashKV) (hdr []string, rows [][]string) {
-	hdr = []string{"endpoint", "hash"}
+	hdr = []string{"endpoint", "hash", "revision"}
```

Comment:
maybe let's be consistent about all presentation forms and call it `hash_revision` ?

Reply:
Makes sense & updated.